librubyfmt/build.rs

```diff
@@ -62,10 +62,10 @@ fn make_configure(ruby_checkout_path: &Path) -> Output {
             .status()?;
         check_process_success("make configure", o)
     } else {
-        let o = Command::new("autoconf")
```

Comment:
Does `autoreconf -i` behave the same as `make configure` if the inputs are unchanged? If so we can probably remove the `if` here

Reply:
Good call. I'm not sure what `make configure` does differently but it's not mentioned in ruby's docs on building so I'm inclined to delete it.

Reply:
It's literally just

```
configure: configure.ac
    autoconf
```